thewalrus/__init__.py

```diff
@@ -68,7 +76,7 @@
     hermite_multidimensional
     reduction
     version
-
+    low_rank_hafnian
```

Comment:
This is a pure python reference implementation, so doesn't really fit under the heading 'Python wrappers'. Maybe worth changing the heading?

Reply:
Good idea. Another option is to just have the function sit in `__init__.py`?

Reply:
Best to keep it in its own file, from experience, best to keep `__init__.py` as small as possible :)

Reply:
OK, I split the header between Python wrappers and pure python functions. Let me know if this is good.

Reply:
:+1: